CHANGELOG

```diff
@@ -0,0 +1,77 @@
+CHANGELOG file
+--------------
+
+The rules for CHANGELOG file:
+
+- entries are sorted newest-first
+- summarize sets of changes (do not reproduce every git log comment here).
+- do not ever delete anything
+- keep the format consistent (88 char width, Y/M/D date format) and do not use tabs but
+  use spaces for formatting
+
+.. inclusion-marker-changelog-start
+
+0.1.5 (XXXX/XX/XX)
+------------------
+
+- Add this CHANGELOG file (#)
```

Comment:
We are about to make a new release. These are the new changes that I would identify as relevant
```suggestion
- Rename RidgeRegression2FoldCV -> Ridge2FoldCV (#211)
- Adding metrics for prediction rigidity (#209)
- Overhaul of documentation page (#200 to #204)
- Add this CHANGELOG file (#198)
```